pkg/apis/app/v2/kieapp_types.go

```diff
@@ -151,7 +151,7 @@ type KieServerSet struct {
 	Kafka                  *KafkaExtObject               `json:"kafka,omitempty"`
 	KafkaJbpmEventEmitters *KafkaJBPMEventEmittersObject `json:"kafkaJbpmEventEmitters,omitempty"`
 	Cors                   *CORSFiltersObject            `json:"cors,omitempty"`
-	// MDBMaxSession number of KIE Executor sessions
+	// MDBMaxSession number of max KIE Executor session, it must be lower than the value of max-pool-size passed as jboss.mdb.strict.max.pool.size, if leaved empty if leaved empty the max.pool.size will be set to 60
```

Comment:
@spolti wdyt about this suggested description?
```suggestion
	// MDBMaxSession number of max KIE Executor session, it must be lower than the value of max-pool-size, by default is max-pool-size set to 60. Max pool size can be set by system property jboss.mdb.strict.max.pool.size (using environment variable JAVA_OPTS_APPEND).
```

Reply:
Added "MDBMaxSession number of max KIE Executor sessions, it must be lower than the value of max-pool-size, by default is max-pool-size set to 60. Max pool size can be set by system property jboss.mdb.strict.max.pool.size (using javaOptsAppend: "-Djboss.mdb.strict.max.pool.size=40"), for more information see https://access.redhat.com/solutions/2955481."